e2e/testcases/stress_test.go

```diff
@@ -288,6 +290,18 @@ func TestStress100CRDs(t *testing.T) {
 	if err := nt.WatchForAllSyncs(); err != nil {
 		nt.T.Fatal(err)
 	}
+
+	// validate throttling in nomos status
```

Comment:
Validate client-side throttling is disabled for nomos status.

Reply:
done